src/leo-server-app/src/routes/satellite.ts

```diff
@@ -135,7 +125,10 @@ function isSunlit(date: Date, lon: number, lat: number, height: number) {
 }
 
 router.get("/getSatelliteInfo", (req: any, res: any) => {
+  const noradId = req.query.noradId ?? "55098";
```

Comment:
can we make a default ID and then define it there

Reply:
actually can we also just make this another function so it looks like const noradId = setNoradId(req.query.noradId) for readability

Reply:
oh true, nice catch